### PR TITLE
LogRhythm - add commands

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17425

## Description
- Add to LogRhythm REST integration the commands:
    lr-get-hosts
    lr-get-alarm-data
    lr-get-alarm-events
    lr-get-networks
    lr-get-persons
- update test playbook

### CODE REVIEW

**Comment by adi88d** [comment]
@kirbles19 please review the descriptions for those commands:
lr-get-hosts
lr-get-alarm-data
lr-get-alarm-events
lr-get-networks
lr-get-persons

**Comment by kirbles19** [comment]
Done editing the .yml.

File: Integrations/LogRhythmRest/LogRhythmRest.yml [hunk 1]
```diff
@@ -657,20 +657,20 @@ script:
...
+      description: The alarm ID.
       type: String
   isfetch: false
   runonce: false
   script: '-'
   type: python
 tests:
 - LogRhythm REST test
-releaseNotes: add lr-get-alarm-data and lr-get-alarm-events commands
+releaseNotes: Added the lr-get-hosts, lr-get-alarm-data, lr-get-alarm-events, lr-get-networks, and lr-get-persons commands.
```
Comment (yaakovi) [inline Integrations/LogRhythmRest/LogRhythmRest.yml]: @adi88d 
pull from master, move the releaseNotes to a file and remove it from here.

File: Integrations/LogRhythmRest/LogRhythmRest.yml [hunk 2]
```diff
@@ -407,9 +407,270 @@ script:
...
+      type: String
+    - contextPath: Logrhythm.Network.ID
+      description: Network ID
+      type: String
+    - contextPath: Logrhythm.Network.EntityId
+      description: Entity ID
+      type: String
+  - arguments:
+    - default: false
+      description: The Log Rhythm host ID.
```
Comment (yaakovi) [inline Integrations/LogRhythmRest/LogRhythmRest.yml]:
```suggestion
      description: The LogRhythm host ID.
```

File: Releases/LatestRelease/LogRhythmRest.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
 - Added 2 new commands:
```
Comment (yaakovi) [inline Releases/LatestRelease/LogRhythmRest.md]: update RN (join the lists togethers

**Pull request merged**